scattr/workflow/rules/zona_bb_subcortex.smk

```diff
@@ -74,7 +74,7 @@ rule reg2native:
         target=lambda wildcards: expand(
             inputs["T1w"].path,
             zip,
-            **filter_list(inputs["T1w"].input_zip_lists, wildcards)
+            **filter_list(inputs["T1w"].zip_lists, wildcards)
```

Comment:
I think we could make this `target=lambda wildcards: inputs["T1w"].filter(**wildcards).expand()`